src/core/Toast/Toast.tsx

```diff
@@ -132,20 +163,50 @@ export const Toast = (props: ToastProps) => {
     }
   };
 
+  const calculateOutAnimation = (node: HTMLElement) => {
+    // calculation translate x and y pixels.
+    let translateX = 0;
+    let translateY = 0;
+    if (animateOutTo && node) {
+      const { x: startX, y: startY } = node.getBoundingClientRect(); // current element
+      const { x: endX, y: endY } = animateOutTo.getBoundingClientRect(); // anchor point
+      translateX = endX - startX;
+      translateY = endY - startY;
+    }
+    return { translateX: translateX, translateY: translateY };
+  };
+
   return (
-    <Transition
-      timeout={{ enter: 240, exit: 120 }}
+    <CSSTransition
+      timeout={{ enter: 240, exit: animateOutTo ? 400 : 120 }}
       in={visible}
       appear={true}
       unmountOnExit={true}
       onExited={onRemove}
```

Comment:
Can you move it bellow `onExited`?  Just for better visibility of what is happening after what.